vllm/config/compilation.py

```diff
@@ -560,6 +561,19 @@ def set_splitting_ops_for_v1(self):
                 self.cudagraph_mode = CUDAGraphMode.FULL
             self.splitting_ops = []
 
+        if envs.VLLM_ALL2ALL_BACKEND == "deepep_high_throughput":
+            # exclude MoE dispatch/combine from capture by ensuring
+            # piecewise splitting includes them, so communication remains
+            # outside CUDA graphs while compute can still be graphed.
+            moe_split_ops = [
+                "vllm.model_executor.layers.fused_moe.fused_moe",
+                "vllm.model_executor.layers.fused_moe.fused_experts",
+                "vllm.moe_forward",
```

Comment:
Are these all supposed to be the names of `torch.ops`?

Reply:
Fixed, thanks!